apps/src/applab/designMode.js

```diff
@@ -1164,7 +1166,7 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
   );
   var children = $(levelDom).children();
   children.each(function() {
-    designMode.parseScreenFromLevelHtml(this, allowDragging, prefix);
+    designMode.parseScreenFromLevelHtml(this, allowDragging, prefix, true);
```

Comment:
NIT: add a comment for the boolean parameter:
 `designMode.parseScreenFromLevelHtml(this, allowDragging, prefix, true /* skipUnknownElements */)`